themes/mcf/pages/homepage.tsx

```diff
@@ -15,15 +19,23 @@ type TProps = {
 
 const LandingPage = ({ handleSearchInput, searchInput }: TProps) => {
   return (
-    <main id="main" className="relative h-full">
-      <div className="bg-cclw-dark text-white">
-        <Hero handleSearchInput={handleSearchInput} searchInput={searchInput} />
-      </div>
-      <div id="world-map" className="container hidden pt-6 md:block">
-        <h2 className="text-center mb-6">Explore by country</h2>
-        <WorldMap />
-      </div>
-    </main>
+    <Layout title="Law and Policy Search">
```

Comment:
Probably bets to mention projects rather than Laws and Policies, something like "Multilateral climate fund search"